File: BackEnd/ajax/Paramedico.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'idPais': null,
    'noIdentidad': null,
    'pNombre': null,
    'sNombre': null,
    'sApellido': null,
    'direccion': null,
    'correo': null,
    'pApellido': null,
    'sexo': null,
    'licencia': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorApellido',
    'sApellido': null,
    'pApellido': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorNoIdentidad',
    'noIdentidad': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

// $.ajax({
//   url:CONST_SITIO_URL+'/services/Paramedico.php',
//   method:'POST',
//   dataType:'JSON',
//   data:{
//     'accion':'buscarPorLicencia',
//     'licencia': null,
  
//   },
//   success:function(respuesta){
//     console.log(respuesta);
//   },
//   error: function(error){
//     console.log(error);
//   },
//   complete: function(){
//     //TO-DO
//   }
// });

$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
    'idCentroMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'correo': null,
    'idParamedico': null,
    'licencia': null,
    'direccion': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorNombre',
    'pNombre': null,
    'sNombre': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Paramedico.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listar',
    'idParamedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});

File: BackEnd/postman/Paramedico.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Paramedico'
data={
  'accion':'crear',
  'idPais': None,
  'noIdentidad': None,
  'pNombre': None,
  'sNombre': None,
  'sApellido': None,
  'direccion': None,
  'correo': None,
  'pApellido': None,
  'sexo': None,
  'licencia': None,
}
print postear(data,url)
print

url='Paramedico'
data={
  'accion':'buscarPorApellido',
  'sApellido': None,
  'pApellido': None,
}
print postear(data,url)
print

url='Paramedico'
data={
  'accion':'buscarPorNoIdentidad',
  'noIdentidad': None,
}
print postear(data,url)
print

url='Paramedico'
data={
  'accion':'listarTodos',
  'idCentroMedico': None,
}
print postear(data,url)
print

url='Paramedico'
data={
  'accion':'actualizar',
  'correo': None,
  'idParamedico': None,
  'licencia': None,
  'direccion': None,
}
print postear(data,url)
print

url='Paramedico'
data={
  'accion':'buscarPorNombre',
  'pNombre': None,
  'sNombre': None,
}
print postear(data,url)
print

url='Paramedico'
data={
  'accion':'listar',
  'idParamedico': None,
}
print postear(data,url)
print
